deposit/src/test/java/edu/unc/lib/deposit/fcrepo4/IngestContentObjectsJobTest.java

```diff
@@ -79,10 +85,21 @@
 	private FolderObject destinationObj;
 
 	@Mock
+	private PremisLoggerFactory premisLoggerFactory;
```

Comment:
What do you think about prepending these two variable names with "mock", just for consistency?